src/index.html

```diff
@@ -17,6 +29,89 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="logo">
+          <a
+            href="index.html"
+            class="logo__link"
+          >
+            <img
+              src="./images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="index.html"
+                class="nav__link is-active under-line"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
            </li>
            
            <li class="nav__item">
```